core/src/main/scala/scalaz/StrictTree.scala

```diff
@@ -170,9 +153,8 @@ case class StrictTree[A](
     *
     * @return
     */
-  override def hashCode(): Int = {
-    runBottomUp(hashCodeReducer)
-  }
+  override def hashCode(): Int =
+    MurmurHash3.listHash(rflatten, "StrictTree".hashCode)
```

Comment:
unclear how ok this is, the hashcode is invariant in tree shape, may reconsider

Reply:
what if we just delete this and `equals`?